tests/improver-estimate-emos-coefficients/02-gaussian.bats

```diff
@@ -37,8 +37,8 @@
 
   # Estimate the EMOS coefficients and check that they match the kgo.
   run improver estimate-emos-coefficients 'gaussian' '20170605T0300Z' \
-      "$IMPROVER_ACC_TEST_DIR/ensemble-calibration/gaussian/history/*.nc" \
-      "$IMPROVER_ACC_TEST_DIR/ensemble-calibration/gaussian/truth/*.nc" \
+      "$IMPROVER_ACC_TEST_DIR/apply-emos-coefficients/gaussian/history/*.nc" \
+      "$IMPROVER_ACC_TEST_DIR/apply-emos-coefficients/gaussian/truth/*.nc" \
```

Comment:
Why are all the estimate-emos-coefficients tests using apply-emos-coefficients folder?

Reply:
Good point. I've updated the CLI tests and moved around the directories containing the historical forecasts and truth inputs, so that these are within the `estimate-emos-coefficients` directory.